ui-v2/tests/integration/components/consul-node-list-test.js

```diff
@@ -0,0 +1,26 @@
+import { module, skip } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | Component | consul-node-list', function(hooks) {
+  setupRenderingTest(hooks);
+
+  skip('it renders', async function(assert) {
+    // Set any properties with this.set('myProperty', 'value');
+    // Handle any actions with this.set('myAction', function(val) { ... });
+
+    await render(hbs`<ConsulNodeList />`);
+
+    assert.equal(this.element.textContent.trim(), '');
+
+    // Template block usage:
+    await render(hbs`
+      <ConsulNodeList>
+        template block text
+      </ConsulNodeList>
+    `);
+
+    assert.equal(this.element.textContent.trim(), 'template block text');
+  });
+});
```

Comment:
I'm thinking moving forwards we could just delete/not add any tests for partial-like components, what do you think? Let's chat next time we hook up.